rmc/pipeline/regional_constraint.py

```diff
@@ -256,25 +269,46 @@ def main(args):
             hl.init(
                 log=f"/round{args.search_num}_single_break_search.log",
                 tmp_dir=TEMP_PATH_WITH_FAST_DEL,
+                quiet=args.quiet,
             )
+            run_single_search = True
 
             logger.info(
                 "Searching for transcripts or transcript subsections with a single significant break..."
             )
             if args.search_num == 1:
-                # Read constraint_prep resource HT if this is the first search
-                ht = constraint_prep.ht()
 
-                logger.info(
-                    "Adding section annotation before searching for first break..."
+                all_loci_chisq_ht_path = (
+                    f"{SINGLE_BREAK_TEMP_PATH}/round{args.search_num}_all_loci_chisq.ht"
```

Comment:
This file is currently saved as just `all_loci_chisq.ht`, do we want to change the name or just remove the `round{args.search_num}` here?

Reply:
hmmmm I'll remove ```round{args.search_num}```. I'll probably delete this HT (and regenerate since this will contain sections with min exp mis < 16